index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="CSS/style.css">
+    <title>Abiy Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <ul>
+            <li><a href="#"><img src="Images\MyLogo.svg" alt="" width="76px" height="20px"></a></li>
+            <li>
+               <nav><img src="Images\menu.svg" alt="" width="18px" height="12px"></nav>
+            </li>
+         </ul>
+    </header>
+    <!-- <img src="../Images/my_background.svg" alt="my background image"> -->
+    <section class="mobile-landing">
+        <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see</span> <span class="wrapper">you!</span></h1>
+        <p>
+            <span class="p-wrapper">I'm a software developer! I can help you</span> 
+            <span class="p-wrapper">build a product , feature or website Look</span> 
+            <span class="p-wrapper">through some of my work and</span> <span></span> 
+            <span class="p-wrapper">experience! If you like what you see and</span> 
+            <span class="p-wrapper">have a project you need coded, don't</span>
+            <span class="p-wrapper">hestiate to contact me.</span>
+         </p>
+         <div class="lets-connect">LET'S CONNECT</div>
+         <div class="lets-connect-icons">
+            <ul>
+               <li><a href="#"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
+               <li><a href="#"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
+               <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
+               <li><a href="#"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
+               <li><a href="#"><img src="Images\Vector5.svg" alt="" width="14.17px" height="20px"></a></li>
```

Comment:
- [ Required ] Good job implementting the social media list correctly, however, I've noticed that the `href` attribute for `GitHub`, `Twitter` and `LinkedIn` are empty yet the readme has these accounts present. It would be better to add the links for GitHub, Twitter and LinkedIn to enable other social media users connect with you.